Update Porkbun

kmpoppe opened the pull request:
Closes #3818.

kmpoppe committed:
Update porkbun

kmpoppe committed:
Change documentation link

This documentation link gets to the category page where there are two articles (19 and 119) that now describe both variants.

Carlgo11 commented on _data/domains.yml:
```diff
@@ -461,7 +461,8 @@ websites:
       software: Yes
-      doc: https://porkbun.com/products/two_factor_authentication
+      hardware: Yes
+      doc: https://kb.porkbun.com/category/20-accounts
```
I think article 19 fits better as it describes how to enable 2FA.
```suggestion
      doc: https://kb.porkbun.com/article/19-how-to-enable-two-factor-authentication
```

Carlgo11 committed:
Update _data/domains.yml

Use article rather than category as per Carl's request.

Co-Authored-By: kmpoppe <kmpoppe@users.noreply.github.com>

jamcat22 committed:
Update Porkbun

- Updated name to use title case.
- Updated doc URL to use .../[ID]-- method for HelpScout-based doc link.

RichJeanes commented on _data/domains.yml:
```diff
@@ -456,12 +456,13 @@ websites:
       software: Yes
-      doc: https://porkbun.com/products/two_factor_authentication
+      hardware: Yes
+      doc: https://kb.porkbun.com/article/19--
```
[Article 19](https://kb.porkbun.com/article/19--) only describes how to set up Software 2FA, but [Article/Category 20](https://kb.porkbun.com/category/20--) links to articles for BOTH [Software](https://kb.porkbun.com/article/19--) and [Hardware](https://kb.porkbun.com/article/119--) 2FA. Preference? @Carlgo11 @kmpoppe @jamcat22 @dsoegijono

kmpoppe replied:
```suggestion
      doc: https://kb.porkbun.com/category/20--
```

kmpoppe committed:
Change documentation URL

Change documentation URL as it describes both software and hardware
